src/Illuminate/Collections/functions.php

```diff
@@ -2,6 +2,47 @@
 
 namespace Illuminate\Support;
 
+use Illuminate\Database\Eloquent\Model;
+
+if (! function_exists('Illuminate\Support\is_model')) {
+    /**
+     * Determine if the given value is a model.
+     *
+     * @internal
+     *
+     * @template TModel
+     *
+     * @param  TModel  $model
+     * @return bool
+     */
+    function is_model($model)
+    {
+        return $model instanceof Model;
+    }
```

Comment:
What exactly is the benefit of this helper? That people do not need to know the class?